.github/workflows/github-pages.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy Documentation to Github Pages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+          submodules: recursive
```

Comment:
We don't have any submodules. So, this can be skipped (I guess).

Reply:
@sobolevn did you test this branch?

Reply:
Not yet.